### DISCUSSION TITLE
Webm not installing

### DISCUSSION THREAD

**Discussion by SossyBakey** [KadeArchive/Kade-Engine#2137] [Q&A]
how do i install webm it saying "Could not clone Git repository" when i install everything correctly

**Comment by NK-523** [comment]
run `haxelib list` in command prompt, paste what appears in the reply.

**Reply by SossyBakey** [reply]
> 
> 
> run `haxelib list` in command prompt, paste what appears in the reply.
actuate: [1.8.9]
box2d: [1.2.3]
discord_rpc: [git]
flixel-addons: [2.10.0]
flixel-demos: [2.7.4]
flixel-templates: [2.6.6]
flixel-tools: [1.5.1]
flixel-ui: [2.3.3]
flixel: [4.9.0]
hscript: [2.4.0]
hxcpp: [4.2.1]
layout: [1.2.1]
lime-samples: [7.0.0]
lime: [7.9.0]
linc_luajit: [0.0.2]
newgrounds: [1.1.4]
openfl-samples: [8.7.0]
openfl-webm: [0.0.4]
openfl: [9.1.0]
polymod: [git]

this is what appeared

**Reply by NK-523** [reply]
actuate: [1.8.9]
box2d: [1.2.3]
discord_rpc: [git]
extension-webm: [git]
faxe: [git]
flixel-addons: [2.10.0]
flixel-demos: [2.7.4]
flixel-templates: [2.6.6]
flixel-tools: [1.5.1]
flixel-ui: [2.3.3]
flixel: [4.9.0]
hscript: [2.4.0]
hxcpp: [4.2.1]
hxvm-luajit: [git]
layout: [1.2.1]
lime-samples: [7.0.0]
lime: [7.9.0]
linc_luajit: [git]
newgrounds: [1.1.4]
openfl-samples: [8.7.0]
openfl: [9.1.0]
polymod: [git]
systools: [1.1.0]
what I got, try reinstalling all of the ones that say git for me.

**Reply by NK-523** [reply]
and make sure you have [git_scm](https://git-scm.com/downloads)
